### DISCUSSION TITLE
view metrics  RI Violations vertipaq analyser

### DISCUSSION THREAD

**Discussion by arnohoedelmans** [DaxStudio/DaxStudio#498] [Q&A] [answered]
Hello,

When i check my modell in vertipaq analyser i get some ri violations. 
I went looking for te problem and my conclusion was that if between two tables there is a many to one relation, every field in the many table should have a value. In my modell not all fields have a value so there a blanks. This causes to show a ri violation. 

What is the impact of this and how should i solve this issue. 

thanx for any help

**Comment by marcosqlbi** [comment, marked as answer]
This is not a problem. The presence of RI violations generates a blank row on the one-side of the relationship and in certain conditions, this could have an impact on performance. However, you should not be worried about that. It is just a warning in case you have RI violations when you don't expect to have them.
In strict dimensional models (star schema, Kimball methodology) you could replace the blank/null with a key corresponding to an existing row in the dimension, but this is not mandatory and could break other parts of the logic of your model.

**Reply by arnohoedelmans** [reply]
Thanx Marco for quick reaction!
